gbmodel/model_sqlalchemy.py

```diff
@@ -3,6 +3,23 @@
 import datetime
 from extensions import db
 from sqlalchemy import exc, func
+from cryptography.fernet import Fernet
+key_file = open("../key.txt")
```

Comment:
should have a try/except here

Reply:
this line is also giving me an error with the current file structure of the repo